packages/cyberstorm/src/components/Toast/ToastContext.tsx

```diff
@@ -0,0 +1,98 @@
+import { createContext, PropsWithChildren, useReducer } from "react";
+import { ToastContainer } from "./ToastContainer";
+import { v4 as uuid } from "uuid";
+import * as RadixToast from "@radix-ui/react-toast";
+
+const initState: {
+  toasts: {
+    id: string;
+    variant?: "info" | "danger" | "warning" | "success";
+    icon?: JSX.Element;
+    message?: string;
```

Comment:
Why does the message change here from `ReactNode` to `string` and from required to optional?

Reply:
Mishap